### PR TITLE
BM Admin Adjustments: Party Map creation flow

### PR DESCRIPTION
**Pull request by yurii-lubynets**
closes https://github.com/sparkletown/internal-sparkle-issues/issues/956
closes sparkletown/internal-sparkle-issues#959
closes sparkletown/internal-sparkle-issues#958
closes sparkletown/internal-sparkle-issues#957

### CODE REVIEW

**Review by malaikaaiyar — APPROVED** [review]
If you've tested it and it works fine, it looks good to me

File: src/pages/Admin/Venue/DetailsForm.tsx [hunk 1]
```diff
@@ -844,157 +577,38 @@ const DetailsFormLeft: React.FC<DetailsFormLeftProps> = ({
...
+          "You can change anything except for the name of your venue later"
+        )}
 
         {renderVenueNameInput()}
         {renderTaglineInput()}
         {renderDescriptionInput()}
-        {renderRestrictToAdultsInput()}
 
         {renderBannerPhotoInput()}
+        {renderHelper("1:1 ratio recommended")}
```
Comment (denisdimitrov) [inline src/pages/Admin/Venue/DetailsForm.tsx:605]: This should be below the logo, not the banner.

File: src/pages/Admin/Venue/DetailsForm.tsx [hunk 2]
```diff
@@ -844,157 +577,38 @@ const DetailsFormLeft: React.FC<DetailsFormLeftProps> = ({
...
-        placeholder="Radio station URL..."
-      />
-      {errors.radioStations && (
-        <span className="input-error">{errors.radioStations.message}</span>
-      )}
-    </div>
+  const renderHelper = (text: string) => (
+    <p
+      className="small light"
+      style={{ marginBottom: "2rem", fontSize: "16px" }}
```
Comment (denisdimitrov) [inline src/pages/Admin/Venue/DetailsForm.tsx:583]: Use classes and avoid inline styles.

File: src/settings.ts [hunk 3]
```diff
@@ -235,18 +235,6 @@ export interface Template_v2 {
...
-  {
-    template: VenueTemplate.zoomroom, // keeping as zoom room for backward compatibility
-    name: "Experience",
-    description: [
-      "Ideal for performances, debates, interactive sessions of all kinds: a Zoom room with its own spot on the Jam",
-    ],
-  },
   {
     template: VenueTemplate.partymap,
     name: "Party Map",
```
Comment (denisdimitrov) [inline src/settings.ts:240]: This and all other text occurrences of `party map` should be renamed to `camp`

**Review by denisdimitrov — CHANGES_REQUESTED** [review]

File: src/pages/Admin/Venue/DetailsForm.tsx [hunk 4]
```diff
@@ -533,16 +434,16 @@ const DetailsFormLeft: React.FC<DetailsFormLeftProps> = ({
...
         image={values.logoImageFile}
         remoteUrlInputName={"logoImageUrl"}
         remoteImageUrl={values.logoImageUrl}
         name={"logoImageFile"}
-        containerClassName="input-square-container"
-        imageClassName="input-square-image"
+        containerClassName="host-icon-container"
+        imageClassName="host-icon"
         error={errors.logoImageFile || errors.logoImageUrl}
       />
```
[the remark this answers is not in the document]
Comment (yurii-lubynets) [reply src/pages/Admin/Venue/DetailsForm.tsx:448]: I assume this is the OOS of this PR.
These changes are in `staging` and will be merged when `staging` and `env/burn` will be synced.

File: src/settings.ts (hunk 3 above)
Comment (yurii-lubynets) [reply src/settings.ts]: I was thinking that renaming is only related to the Admin form and fixed it here https://github.com/sparkletown/sparkle/pull/1930/files#diff-ef665bebb841378779cc61dd106ec01e676fb327a19ec2add06edaf05884acd0R348-R351

But your solution looks better! Thank you

Comment (yurii-lubynets) [reply src/settings.ts]: fixed in 345d697daa39b113824de2229eb71bbbe6d4bc28

File: src/pages/Admin/Venue/DetailsForm.tsx (hunk 4 above)
[the remark this answers is not in the document]
Comment (yurii-lubynets) [reply src/pages/Admin/Venue/DetailsForm.tsx:445]: fixed in def392d3865e2f8d0eaf232f49ef621de2d48e48

File: src/pages/Admin/Venue/DetailsForm.tsx (hunk 1 above)
Comment (yurii-lubynets) [reply src/pages/Admin/Venue/DetailsForm.tsx]: fixed in def392d3865e2f8d0eaf232f49ef621de2d48e48

File: src/pages/Admin/Venue/DetailsForm.tsx (hunk 2 above)
Comment (yurii-lubynets) [reply src/pages/Admin/Venue/DetailsForm.tsx]: fixed in def392d3865e2f8d0eaf232f49ef621de2d48e48

**Review by denisdimitrov — APPROVED** [review]
LGTM!

**Comment by whoaitsaimz** [comment]
This was a heck of a lot of code to delete from our admin (rather than just commenting out). For example, toggle functionality is now completely stripped, so it will be more difficult to add any new admin toggle to this environment, won’t it?

**Comment by yurii-lubynets** [comment]
> This was a heck of a lot of code to delete from our admin (rather than just commenting out). For example, toggle functionality is now completely stripped, so it will be more difficult to add any new admin toggle to this environment, won’t it?

Yes, it could be. But it was an update in the scope of the task list described above https://github.com/sparkletown/sparkle/pull/1930#issue-697904414

**Pull request merged**